### PR TITLE
feat(`NcListItem`): introduce 'details' slot

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Provide an option to use slot (for embedding icon / custom template) instead of prop string

### 🖼️ Screenshots

Screenshot from 2024-01-26 12-31-55

### 🚧 Tasks

- [ ] Design review
- [ ] Code review

### 🏁 Checklist

- [ ] ⛑️ Tests are included or are not applicable
- [x] 📘 Component documentation has been extended, updated or is not applicable

### CODE REVIEW

**Review by marcoambrosini — APPROVED** [review]
Nice :)

File: src/components/NcListItem/NcListItem.vue [hunk 1]
```diff
@@ -87,14 +87,16 @@
...
-			:details="'1h'"
 			:counter-number="44">
 			<template #icon>
 				<NcAvatar :size="44" user="janedoe" display-name="Jane Doe" />
 			</template>
 			<template #subname>
 				In this slot you can put both text and other components such as icons
 			</template>
+			<template #details>
+				<LinkIcon :size="16"/>
```
Comment (susnux) [inline src/components/NcListItem/NcListItem.vue:98]: should it not be 20 as that's our default?

**Review by susnux — APPROVED** [review]

File: src/components/NcListItem/NcListItem.vue (hunk 1 above)
Comment (Antreesy) [reply src/components/NcListItem/NcListItem.vue:98]: Same as bubble icon below, and close to the font size. But that's up to end developer anyway, I believe

**Pull request merged**